src/cycax/cycad/features.py

```diff
@@ -148,14 +149,10 @@ def export(self)-> dict:
             dict: this will return a dictionary.
 
         """
-        dict_nut = {
-            "name": "nut",
-            "type": "cut",
-            "side": self.side,
-            "x": str(self.x),
-            "y": str(self.y),
-            "z": str(self.z),
-            "diameter": self.type,
-            "depth": self.depth,
-        }
+        dict_nut = {}
+        for key, value in vars(self).items():
+            dict_nut[key] = value
```

Comment:
This is exactly the same as `dict_nut = vars(self)`, but leave it like this since we will add a filter in here. e.g.
```
for key, value in vars(self).items():
    if not key.startswith('_'):
         dict_nut[key] = value
```